access/types/stdsync.py

```diff
@@ -131,7 +131,7 @@ def createForm(request, course, exercise, post_url):
         last = max_n > 0 and n >= max_n
 
     form = GradedForm(request.POST or None, request.FILES or None,
-        exercise=exercise, show_correct_once=last)
+        exercise=exercise, show_correct_once=last, request=request)
```

Comment:
The `GradedForm` constructor is also used in other places in the code and the request argument should be added there as well if applicable.